app/transport/http/server.go

```diff
@@ -2,26 +2,31 @@ package http
 
 import (
 	"fmt"
+	"github.com/go-devs-ua/octagon/cfg"
+	"github.com/gorilla/mux"
 	"net/http"
+	"time"
 )
```

Comment:
There is a rule of the imports placement. The standard libraries are on the top part of import and the parts of imports are separated by an empty line, defining groups of imports